mssqlcli/mssql_cli.py

```diff
@@ -179,14 +176,15 @@ def __init__(self, options):
             self.query_history = []
 
             # Initialize completer
-            smart_completion = True if c['main'].get('smart_completion', 'True') == 'True' else False
-            self.completer = MssqlCompleter(smart_completion=smart_completion, settings=self.settings)
+            smart_completion = c['main'].get('smart_completion', 'True') == 'True'
```

Comment:
Minor: is there any case where 'true' would be coming back here? Case-sensitive comparisons are always liable to have issues

Reply:
Great point, it's unlikely but I don't think it hurts to throw in a `'.lower()`.